digital_identity_client.go

```diff
@@ -76,3 +76,8 @@ func (client *DigitalIdentityClient) GetShareSession(sessionID string) (*digital
 func (client *DigitalIdentityClient) CreateShareQrCode(sessionID string) (share *digitalidentity.QrCode, err error) {
 	return digitalidentity.CreateShareQrCode(client.HTTPClient, sessionID, client.GetSdkID(), client.getAPIURL(), client.Key)
 }
+
+// Get session QR code based on generated Qr ID
+func (client *DigitalIdentityClient) GetQrCode(qrCodeId string) (share digitalidentity.ShareSessionQrCode, err error) {
```

Comment:
```suggestion
// GetShareSessionQrCode retrieves a session QR code based on generated Qr ID
func (client *DigitalIdentityClient) GetShareSessionQrCode(qrCodeId string) (share digitalidentity.ShareSessionQrCode, err error) {
```

comment + func name (i haven't checked other sdks so happy to just have it all align with whatever they have)